server/models/appointment.ts

```diff
@@ -1,12 +1,8 @@
-import AppointmentAttendance from './appointmentAttendance'
-import AppointmentBehaviour from './appointmentBehaviour'
+import SessionFeedback from './sessionFeedback'
 
 export default interface Appointment {
```

Comment:
Do you think it could be clearer to call this `SupplierAssessmentAppointment` to distinguish it from `ActionPlanAppointment` (as they're both sort of appointments), or is the `Appointment` going to be used elsewhere?

Reply:
I think that `Appointment` is going to become a generic thing that will replace `ActionPlanAppointment`, eventually.